conf/test.config

```diff
@@ -54,6 +54,7 @@ profiles {
         params.step                = 'annotate'
     }
     no_intervals {
+        params.intervals           = null
```

Comment:
do we need to add the same to the nextflow config/ some logic for params.no_intervals?

Reply:
I'm assuming that people specifying `--no_intervals` mostly do that with a custom genome, and don't provide `--intervals` then.
Worst case for people using `--no_intervals` and iGenomes for genomes where we do provide `intervals`, we have just this warning.

Reply:
ok